### PR TITLE
BI-53: linhhoang16 - Printing Invoices

### PR DESCRIPTION
**Pull request by linhhoang16**
## Exercise

* [Exercise-53](https://github.com/NT-Line-team/Behave_test/issues/53)

## Overview

An invoice item has the following main attributes: name, quantity (qty), unit price and amount. Write a Python program that lets user input a list of items. When user finishes data inputting process, the program should print out the invoice as below:
```
Item	Name	Qty	Unit Price	Amount
1	Apple iPhone	2	650	1300
2	Samsung TV	5	300	1500
3	Sony Washing Machine	1	750	750
Amount			3550
VAT (10%)			355
TOTAL AMOUNT			3905
```
Notes

VAT rate is defined at 10%
Items should be ordered by name.
Table and front layouts are NOT in the scope

## Verification Steps
Please input name of product: Samsung TV
Please input quantity: 5
Please input unit price: 300
Do you want to add new item?y
Please input name of product: Apple iPhone
Please input quantity: 2
Please input unit price: 650
Do you want to add new item?y
Please input name of product: Sony Washing Machine
Please input quantity: 1
Please input unit price: 750
Do you want to add new item?n

### CODE REVIEW

File: python_basic/linhhoang16/printing_invoice.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+name_list =[]
+qty_list = []
+price_list = []
+amount_list=[]
+record_list =[]
+sum_amount=0
+vat=0
+total_amount=0
+carryon_run =True
```
Comment (hungphamviet) [inline python_basic/linhhoang16/printing_invoice.py]: From line 1-9, please have spaces around `=` operators

File: python_basic/linhhoang16/printing_invoice.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+while carryon_run:
+    name = raw_input ("Please input name of product: ")
+    qty = raw_input ("Please input quantity: ")
+    unit_price = raw_input ("Please input unit price: ")
+    record_list.append(name)
+    record_list.append(qty)
+    record_list.append(unit_price)
+    record_list.append(int(qty) * int(unit_price))
```
Comment (hungphamviet) [inline python_basic/linhhoang16/printing_invoice.py]: `qty` and `unit_price` will be `int` if you use `input` method. In that case, no need to cast it into `int` type.

File: python_basic/linhhoang16/printing_invoice.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    name_list.append(record_list)
+    record_list=[]
+    carryon= raw_input ("Do you want to add new item?")
+    if carryon=="n":carryon_run= False
+for i in range (len(name_list)):
+    sum_amount += name_list[i][3]
```
Comment (hungphamviet) [inline python_basic/linhhoang16/printing_invoice.py]: To be better code...
```
for name in name_list:
   sum_amount += name[3]
```

File: python_basic/linhhoang16/printing_invoice.py [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+vat = sum_amount * 0.1
+total_amount = sum_amount +vat
+new_name_list = sorted(name_list, key=lambda k: k[0])
```
Comment (hungphamviet) [inline python_basic/linhhoang16/printing_invoice.py]: :100:

File: python_basic/linhhoang16/printing_invoice.py [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+carryon_run =True
+while carryon_run:
+    name = raw_input ("Please input name of product: ")
+    qty = raw_input ("Please input quantity: ")
+    unit_price = raw_input ("Please input unit price: ")
+    record_list.append(name)
+    record_list.append(qty)
+    record_list.append(unit_price)
+    record_list.append(int(qty) * int(unit_price))
+    name_list.append(record_list)
```
Comment (hungphamviet) [inline python_basic/linhhoang16/printing_invoice.py]: Why it is `name_list`? It contains names only?

**Comment by linhhoang16** [comment]
Hi anh @hungphamviet ,

I already update the code as per your comments. Please hep to review again.

Many thanks,
Linh

**Pull request merged**